ipalib/frontend.py

```diff
@@ -1041,18 +1043,23 @@ def output_for_cli(self, textui, output, *args, **options):
 
         self.log_messages(output)
 
-        order = [p.name for p in self.output_params()]
+        order = deque()
+        labels = dict()
```

Comment:
```
labels = {}
flags = {}
```

It's shorter, easier to read, and faster.

Reply:
Okay.